website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ce-chart.md

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
Sorry, one other thing, can we change the file name here please? Can we include the v2 part somewhere. Currently it reads like it's about migrating to CE chart... from EE chart may be? The title of this page is nice, let's make the URL read similar. 🙂